tools/c7n_index/c7n_index/metrics.py

```diff
@@ -153,11 +161,23 @@ def index(self, points):
             p['_index'] = self.config['indexer']['idx_name']
             p['_type'] = self.es_type
 
-        results = helpers.streaming_bulk(self.client, points)
+        results = elasticsearch.helpers.streaming_bulk(self.client, points)
         for status, r in results:
             if not status:
                 log.debug("index err result %s", r)
 
+    def index_sqs(self, queue_msg):
+        try:
+            res = self.client.index(
+                index=self.config.get('idx_name','c7n'),
+                doc_type=self.config.get('doc_type', 'c7n'),
```

Comment:
hmm... this seems very odd usage of elasticsearch, its going to default all the resources to the same index and the same doc type? its just indexing the raw body of the message? that seems less than useful... your getting a big index with no fielded search capability outside of policy spec, and very little search outside of at best bag of words on the resources.

Reply:
Our initial recommendation was to do this because `_type` is deprecated and is being removed in the next ES release so at this point, it doesn't seem to matter what you set it to.

Reply:
Thanks, i guess the point i was making is i'm not seeing any evidence here of thinking about the data model, generally mapping data to nosql and indexed stores requires careful consideration of usage/query patterns. This would pr would be a bit better with some examples of the queries being considered. 

re type removal this is a good read https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html